Tests/PKHeX.Core.Tests/Legality/LearnabilityTests.cs

```diff
@@ -58,4 +59,49 @@ public static void VerifyCanLearnTM(string species, params string[] moves)
         var can = EncounterLearn.CanLearn(species, moves);
         can.Should().BeTrue($"{species} should be able to learn all moves: {string.Join(", ", moves)}");
     }
+
+    [Theory]
```

Comment:
```suggestion
    [Theory]
    [InlineData(GameVersion.GS, Species.Paras, Move.Counter, Move.Flail, Move.LightScreen)]
```

https://www.smogon.com/forums/threads/bug-report-validator.3774996/
> Magikarp learns Flail by levelup.
> Chainbreeding Flail: Magikarp -> Horsea ->Squirtle -> Chikorita -> Paras -> Heracross.
> Heracross learns Counter by levelup.
> Chainbreeding Flail+Counter: Heracross -> Chikorita.
> Chikorita learns Light Screen by levelup.
> Breeding Chikorita with Light Screen, Counter and Flail with female Paras results in Paras having those moves.